contrib/build_purger.rb

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env ruby
+require 'json'
+require 'time'
+require 'optparse'
+require 'yaml'
+
+options = {}
+OptionParser.new do |opts|
+  opts.banner = "Usage: build_purger.rb [options]"
```

Comment:
We could add a small description here so that the user knows what this is about. Something like:

> Purge old mistry builds from the file system.

Reply:
done